certbot/certbot/_internal/cli/helpful.py

```diff
@@ -410,8 +424,22 @@ def add_deprecated_argument(self, argument_name, num_args):
         :param int nargs: Number of arguments the option takes.
 
         """
-        util.add_deprecated_argument(
-            self.parser.add_argument, argument_name, num_args)
+        # certbot.util.add_deprecated_arguments expects the normal add_argument
+        # interface provided by argparse. This is what is given including when
+        # certbot.util.add_deprecated_arguments is used by plugins, however, in
+        # that case the first argument to certbot.util.add_deprecated_arguments
```

Comment:
Oops, one minor typo in this explainer: `add_deprecated_arguments` -> `add_deprecated_argument`.

Otherwise LGTM.

Reply:
There's 2 other instances in that paragraph (sorry)

Reply:
No worries. Thanks for catching them.